src/styles/scss/shared.scss

```diff
@@ -23,7 +23,18 @@
   overflow: hidden;
 }
 
+.modal-screen-reader-description {
+  position: absolute;
+  clip: rect(1px, 1px, 1px, 1px);
+  padding: 0;
+  border: 0;
+  height: 1px;
+  width: 1px;
+  overflow: hidden;
+}
```

Comment:
Please reconsider this.

As far as I understand this is a variation of the `.hide-visually` but [with additional considerations/improvements regarding accessibility](https://webaim.org/techniques/css/invisiblecontent/#offscreen). If that is the case then I suggest that we update the `.hide-visually` class to use this method instead of of having two classes that do the same thing.

If you think there is value in having a named BEM element (although these classes do not follow the BEM naming convention) to use here I would add a placeholder selector `%hide-visually` and `@extend` from that.

Reply:
I'll change this, you are right, I was not aware of the ```.hide-visually``` class.